source/Nuke.Common/NukeBuild.cs

```diff
@@ -65,14 +65,23 @@ namespace Nuke.Common
     [SerializeBuildServerState]
     public abstract partial class NukeBuild : INukeBuild
     {
+        /// <summary>
+        /// Gets or sets the build exit code.
+        /// When set to <value>null</value> (default), <see cref="Execute{T}"/> will return a <em>0</em> exit code on build success; or a <em>-1</em> exit code on build failure.
+        /// When set to a non-null value, <see cref="Execute{T}"/> will return the value of <see cref="ExitCode"/>.
+        /// </summary>
+        protected static int? ExitCode { get; set; }
```

Comment:
Can you move this after `IsSuccessful` and make it `public` (non-static) ?

Reply:
There you go